### PR TITLE
Added Sine Function

### PR DESCRIPTION
**Pull request by AggRag**

### CODE REVIEW

File: Calculator.py [hunk 1]
```diff
@@ -17,3 +18,8 @@ def average(self, nums):
     nums_sum = self.sum(nums)
     result = nums_sum / l
     return result
+  def sine(self,nums):
+    result = []
+    for i in nums:
+        result.append(math.sin(i))
```
Comment (itaditya) [inline Calculator.py]: use Python3 map to generate a list of sines from list of angles. Take a look at this PR https://github.com/itaditya/python-calculator/blob/master/Calculator.py#L22

You won't need lambda though

File: cli.py [hunk 2]
```diff
@@ -19,6 +19,8 @@ def calculate(operation, integers):
     return calc.multiply(integers)
   if (operation == 'average'):
     return calc.average(integers)
+  if (operation == 'sine'):
```
Comment (itaditya) [inline cli.py]:
```suggestion
  if (operation == 'sin'):
```

File: Calculator.py (hunk 1 above)
Comment (AggRag) [reply Calculator.py]: Done

File: cli.py (hunk 2 above)
Comment (AggRag) [reply cli.py]: Done

**Comment by itaditya** [comment]
Resolve the conflicts

**Comment by itaditya** [comment]
Congrats @RaghavAGG12

**Pull request merged**